### PR TITLE
Complete redux store for bookstore project

### PR DESCRIPTION
**Pull request by coderacheal**
- Created a directory that will contain all your Redux logic (/src/redux)
- Configured a Redux store (/src/redux/store.js)
- Defined a slice of state for books that:
- Includes an array of books (initial state: empty array)
- Includes a reducer that adds a book
- Includes a reducer that removes a book
- Define a slice of state for categories that:
- Includes an array of categories (initial state: empty array)
- Includes a reducer that check the status and always return "Under construction"

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @coderacheal 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️ Kindly make sure you have a Readme created with [Microverse Readme Template](https://github.com/microverseinc/readme-template/blob/master/README.md).
Screenshot 2023-03-09 at 10 10 26 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state) => {
+      state.booksList = [];
+    },
+    removeBook: (state) => {
+      state.booksList = [];
+    },
```
Comment (Deepakdanger) [inline src/redux/books/booksSlice.js]: - [ ] I am not seeing any actions here. Please add the code to add and remove the book.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @coderacheal & @huseyinbiyik, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Professional readme file ✔️
- Good project structure ✔️
approval1

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**